File: Chapter 05/Python code/5_20.py
n= int(input("Enter a number"))
for i in range(1,n+1): 
    for k in range(1, i):
	    print(" ", end=" ")
    for j in range(1, n-i+2):
	    print(n-i+1, end=" ")
    for l in range(1, n-i+1):
	    print(n-i+1, end=" ")
    print("\n")

File: Chapter 05/C Code/5_20.c

#include <stdio.h>

int main() {
    int n;
    printf("Enter a number: ");
    scanf("%d", &n);

    for (int i = 1; i <= n; i++) {
        for (int k = 1; k < i; k++) {
            printf(" ");
        }
        for (int j = 1; j <= n - i + 1; j++) {
            printf("%d ", n - i + 1);
        }
        for (int l = 1; l < n - i + 1; l++) {
            printf("%d ", n - i + 1);
        }
        printf("\n");
    }

    return 0;
}
